seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4349,11 +4353,11 @@
       "id": "56533eb9ac21ba0edf2244c9",
       "title": "Accessing Object Properties with Variables",
       "description": [
-        "Another use of bracket notation on objects is to use a variable to access a property. This can be very useful for iterating through lists of the object properties or for doing the lookup.",
+        "Another use of bracket notation on objects is to access a property which is stored as the value of a variable. This can be very useful for iterating through lists of the object properties or for doing the lookup.",
         "Here is an example of using a variable to access a property:",
-        "<blockquote>var someProp = \"propName\";<br>var myObj = {<br>  propName: \"Some Value\"<br >};<br>myObj[someProp]; // \"Some Value\"</blockquote>",
-        "Here is one more:",
-        "<blockquote>var myDog = \"Hunter\";<br>var dogs = {<br>  Fido: \"Mutt\",\n  Hunter: \"Doberman\",\n  Snoopie: \"Beagle\"<br >};<br>var breed = dogs[myDog];<br>console.log(breed);// \"Doberman\"</blockquote>",
+        "<blockquote>var dogs = {<br>  Fido: \"Mutt\",\n  Hunter: \"Doberman\",\n  Snoopie: \"Beagle\"<br>};<br>var myDog = \"Hunter\";<br>var breed = dogs[myDog];<br>console.log(breed); // \"Doberman\"</blockquote>",
+        "Bracket notation is also useful when you have to access a property whose name is collected dynamically during the program execution, as follows:",
```

Comment:
It feels out-of-place to go back to talking about bracket notation on a challenge about accessing values with variables. I would just replace `"Bracket notation is also useful when you have to access a property whose name is collected..."`

to:

`"Another way you can use this concept is when the property's name is collected...as in the following example:"`